Makefile

```diff
@@ -186,6 +186,7 @@ up:
 	$(DOCKER_COMPOSE) up -d --remove-orphans
 
 perm:
+	$(EXEC) chmod 666 var app/migrations/*
```

Comment:
1) what is problem with perms?
2) check juste the next line, which changes the perms of `var`

Reply:
var have to probleme, it's when we create a migration , his owner is root:root ...

Reply:
You already tried to fix that problem in a similar way in the past and we told you it was a bad idea. 

Same thing here, the problem is that the created file should belong to your host user, so `chmod` is not the right solution.

Anyway, just `chown` the new migration when created. If you are so lazy that you don't want to do a second command after the migration is created, just create your own alias/command/script or whatever.

The only solution to me is a correct entrypoint but some of us are not able to get it work properly so we must deal with that sort of little problems: not a big deal so deal with it and let it go.

Reply:
Then we have to remove globality of "perm" rule ?

Reply:
No why are you asking this question ?

`perm` rule is related to things we **must** fix to properly run the app, it's not going further than that.